test/transactionBreakdown.test.ts

```diff
@@ -0,0 +1,57 @@
+import { connectDatabase, disconnectDatabase } from '../utils/test-utils/db.util';
+import supertest from 'supertest';
+import testApp from '../app';
+const request = supertest(testApp);
+
+import transactionBreakdownSeed from './transactionBreakdown.test.data';
+
+describe('transactionBreakdown routes', () => {
+  beforeAll(() => {
+    connectDatabase();
+  });
+
+  afterAll(() => {
+    disconnectDatabase();
+  });
+
+  describe('POST route', () => {
+    it('should return a transactionBreakdown with accurate orderTransaction', async () => {
+      const res = await request.post('/api/transactionBreakdown').send(transactionBreakdownSeed[0]);
+      const { orderTransaction } = res.body;
+      expect(res.status).toBe(200);
+      expect(orderTransaction.people.length).toBe(4);
+      expect(orderTransaction.amountDetail.amount).toBe(17.4);
+      expect(orderTransaction.amountDetail.subAmount).toBe(13.0);
+    });
+
+    it('should return a transactionBreakdown with accurate orderTransactionPersons', async () => {
+      const res = await request.post('/api/transactionBreakdown').send(transactionBreakdownSeed[0]);
+      const { orderTransactionPersons } = res.body;
+
+      expect(res.status).toBe(200);
+      expect(orderTransactionPersons.length).toBe(4);
```

Comment:
let's compare `orderTransactionPersons` with an array of objects with the exact values we're expecting instead of looking at the length of the array and only at one person. same with `orderTransaction`. the reason being is to make sure that the calculations aren't messed up for some but work for others.